backend/test/rules/effects_test.clj

```diff
@@ -58,3 +58,29 @@
           (play-card/play-card 1 1 1)
           (alter-card/alter-card [:rows 0 0] {:power 9001})
           :rows (get 0) (get 0)))))
+
+(defexpect alter-card-relative
```

Comment:
I miss a test when cards have more attributes than `:power`, checking it keeps the other attributes intact.
Also, you are checking the same `path` on the three tests, that doesn't ensure it works as intended.

Reply:
Completely agreed on both accounts